scss/_base_tables.scss

```diff
@@ -71,6 +71,6 @@
   // stylelint-enable selector-max-type
 
   %table-row-border {
-    border-top: 1px solid $colors--theme--border-low-contrast;
+    border-top: 1px solid $colors--theme--border-default;
```

Comment:
I think this needs to stay as `-low-contrast` or otherwise all row borders in all columns become move visible than they are now.